index.yaml

```diff
@@ -1,161 +1,163 @@
 tools:
-  sendgrid:
-    reference: ./sendgrid
-  file-summarizer:
-    reference: ./file-summarizer
-  google-calendar:
-    reference: ./google/calendar
-  database:
-    reference: ./database
-  zoom:
-    reference: ./zoom
-  atlassian-jira:
-    reference: ./atlassian/jira
-  wordpress:
-    reference: ./wordpress
-  excel:
-    reference: ./excel
-  browser:
-    reference: ./browser
-  bluesky:
-    reference: ./bluesky
-  slack:
-    reference: ./slack
-  slack-bot:
-    reference: ./slack/tool-bot.gpt
-  linkedin-publishing:
-    reference: ./linkedin-publishing
-  notion:
-    reference: ./notion
-  google-docs:
-    reference: ./google/docs
-  google-gmail:
-    reference: ./google/gmail
-  google-search:
-    reference: ./google/search
-  google-search-console:
-    reference: ./google/search-console
-  google-sheets:
-    reference: ./google/sheets
-  gitlab:
-    reference: ./gitlab
-  images:
-    reference: ./images
-  outlook-mail:
-    reference: ./outlook/mail
-  outlook-calender:
-    reference: ./outlook/calendar
-  github:
-    reference: ./github
-  time:
-    reference: ./time
-  workspace-files:
-    reference: ./workspace-files
-  tasks:
-    reference: ./tasks
-  threads:
-    reference: ./threads
-  knowledge:
-    reference: ./knowledge
-  shell:
-    reference: ./shell
-  word:
-    reference: ./word
-  tavily:
-    reference: ./search/tavily
-  googlecustomsearch:
-    reference: ./search/google/googlecustomsearch
-  pagerduty:
-    reference: ./pagerduty
-  postgres:
-    reference: ./postgres
-  survey:
-    reference: ./survey
-  weather:
-    reference: ./weather
-  die-roller:
-    reference: ./die-roller
-  smartthings:
-    reference: ./smartthings
-  gemini-image-generator:
-    reference: ./gemini/image-generator.gpt
-  firecrawl:
-    reference: ./firecrawl
-  colly-scraper:
-    reference: ./colly-scraper
-  obot-search:
-    reference: ./search/tavily/obot-search.gpt
-  search-combo:
-    reference: ./search-combo
-  aws-s3:
-    reference: ./aws/s3
-  aws-ec2:
-    reference: ./aws/ec2
-  discord:
-    reference: ./discord
+    sendgrid:
```

Comment:
Seems like you changed the whitespace in this file. It's a best practice to not do so. See if you can add proxycurl here without any other changes to the file